podo/cli/PodoCount/PodoCount.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+from ctk_cli import CLIArgumentParser
+
+def main(args):  
+    
+    cmd = "python3 ../PodoCount_code/podocount_main_serv.py  --type '{}' --input_image '{}' --glom_xml '{}' --basedir '{}' --slider {}  --section_thickness {} --girderApiUrl {} --girderToken {} \
+            ".format(args.type, args.input_image, args.glom_xml, args.basedir, args.slider, args.section_thickness, args.girderApiUrl, args.girderToken)
```

Comment:
I think we can use enumeration for `args.type` to make it more convenient for users to choose. See [here ](https://github.com/girder/slicer_cli_web/blob/458d616f2807b541b955a915cbb23e4575b8e8c3/example-average-color/average_color/average_color.xml#L19). Let's use this type.